src/scripts/interface/cellHighlightRenderer.ts

```
@@ -1,23 +1,50 @@
 import { PieceInfo } from 'domain/piece';
 import { GameInfo } from 'domain/game';
+import { Move } from 'domain/move';
+import { RVector2, Vector2 } from 'math/Vector2';
 
 /**
  * Renders cell highlights
  */
 export class CellHighlightRenderer {
-  private readonly _cellContext = Elements.findById<HTMLCanvasElement>('cell-canvas').getContext('2d')!;
+  private readonly _cellContext =
+    Elements.findById<HTMLCanvasElement>('cell-canvas').getContext('2d')!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)